index.html

```diff
@@ -112,91 +114,128 @@ <h1 style="margin-top: 10px">Smart figures with metadata</h1>
+        <h3>Upgrade the app</h3>
         <p>
+          Stop OMERO.web.
+          To upgrade a version of figure prior to <strong>2.0.1</strong>.
           <ul>
             <li>
-              To update an existing install of OMERO.figure to the latest release, download it,
-              unzip and replace the contents of the current /figure
-              directory with the contents of the unzipped directory.
+              Remove the 'figure' directory from <strong>within</strong> the directory that is on your PYTHONPATH.
             </li>
-            <li>NB: If you are upgrading from OMERO.figure 1.0.0 to 1.1.0 or later,
-                note that we now use Python-Markdown
-                (optional, see install instructions above).</li>
-            <li>Replace PDF generation script (need to be Admin):
-              <code style="width:700px">
-                $ cd figure/scripts   &nbsp &nbsp &nbsp # NB: important to be in this directory<br>
-                # List scripts to get the ID of omero/figure_scripts/Figure_To_Pdf.py<br>
-                $ path/to/bin/omero script list<br>
-                $ path/to/bin/omero script replace &ltscriptId&gt omero/figure_scripts/Figure_To_Pdf.py
+            <li>Install the app using <a href="https://pip.pypa.io/en/stable/">pip</a>:<br>
+              <code style="width:650px">
+                $ pip install omero-figure
               </code>
             </li>
-            <li>If upgrading from OMERO.figure 1.0.0 to 1.1.0 or later, you can update the
+            <li>Remove the old 'figure' link<br>
+                <code style="width:700px">
+                  $ bin/omero config remove omero.web.apps '"figure"'<br>
+                  # Register the app<br>
+                  $ bin/omero config append omero.web.apps '"omero_figure"'<br>
+                </code>
+            </li>
+            <li>If upgrading from OMERO.figure 1.0.0, you can update the
               webclient link to figure,
               so that it automatically opens the figure app in a new tab. Simply remove the old link and add back the
-              new one...
+              new one...<br>
                 <code style="width:700px">
                   # In case you need to list current links...<br>
                   $ bin/omero config get omero.web.ui.top_links<br>
-                  <br>
                   # Remove the figure link...<br>
                   $ bin/omero config remove omero.web.ui.top_links '["Figure", "figure_index"]'<br>
-                  <br>
                   # Add back new link as described in the install instructions above
                 </code>
-                NB: If your OMERO.web version is earlier than 5.1.0 then this change
-                will be ignored until you upgrade.
             </li>
-            <li>Restart web. How you do this will depend on how you have deployed OMERO.web.
+            <li>Restart OMERO.web as normal.
+            </li>
+          </ul>
+          <br>
```

Comment:
This gives a rather huge gap before the following text, maybe try a `<p>` instead?